### DISCUSSION TITLE
How to access $store outside an alpinejs component?

### DISCUSSION THREAD

**Discussion by Jonattb** [alpinejs/alpine#2563] [1. Help] [answered]
Hello! I have an store defined, but i need to modified it inside a socketio, so i need to access to the properties outside an alpinejs component, from windows.Alpines or other way. It is possible?

store.js
```
document.addEventListener('alpine:init', () => {
  Alpine.store('darkMode', {
      init() {
          this.on = window.matchMedia('(prefers-color-scheme: dark)').matches
      },
      on: false,
      toggle() {
          this.on = ! this.on
      }
  })
})
```

application.js
```
console.log( window.Alpine.$store['darkMode'].on ) // undefined
```

Thanks!

**Comment by danddanddand** [comment, marked as answer]
Alpine.store('darkMode').on

**Reply by Jonattb** [reply]
Ohh, i have tried this before and it did not work. but now yes . Thank you!
